### DISCUSSION TITLE
Tagging a new release.

### DISCUSSION THREAD

**Discussion by LiamKearn** [rxhanson/Rectangle#697] [Q&A]
Apologies for the poor medium I wasn't sure where to post it.

Is it possible to tag a new release so those of us who don't have an Apple build chain setup can use the new features/localisations in the newest commits.

If there is a schedule or this isn't possible for a while please let me know and I'll switch to a custom build.

Thanks very much for your time and great software :)

**Comment by rxhanson** [comment]
Thanks for asking and apologies on the delay for getting a release out. My typical release schedule is every 1-2 months depending on how much is committed. My plan is to get a release out first week of January. 

I am finally recovering from an injury that dramatically reduced the amount of time I was capable of spending using a keyboard for the past six months, so I expect to resume normal release timeframes going forward.

**Reply by LiamKearn** [reply]
No worries at all.

I don't know if time away from the keyboard is a positive or negative these days 😄. Either way hope you are doing better soon and thanks again for the support & software.

**Comment by Aaron-Rumpler** [comment]
You could set the CI up to upload the build of each commit as an artifact (retained for 90 days, so you could download a build of any commit within that time period). [Aaron-Rumpler/Rectangle:ci](https://github.com/Aaron-Rumpler/Rectangle/tree/ci) does this currently, and is what I use for testing (I'll get around to installing Xcode at some point).

**Reply by rxhanson** [reply]
Excellent idea. I’ll give this a go when I’m on my next Rectangle sprint.

**Reply by Aaron-Rumpler** [reply]
I've made a PR (#812) of what I've got for that.
